docs/src/examples/components/Icon/Variations/IconExampleCircular.shorthand.tsx

```diff
@@ -3,17 +3,54 @@ import { Icon } from '@stardust-ui/react'
 
 const IconExampleCircular = () => (
   <div>
-    <Icon name="chess rook" circular />
-    <Icon name="book" circular />
-    <Icon name="expand" circular />
-    <Icon name="play" circular />
-    <Icon name="call" circular />
-    <Icon name="call-video" circular />
-    <Icon name="stop" circular />
-    <Icon name="calendar alternate outline" circular />
-    <Icon name="coffee" circular />
-    <Icon name="compass outline" circular />
-    <Icon name="area chart" circular />
+    <Icon
+      name="chess rook"
+      circular
+      variables={siteVars => ({
+        backgroundColor: siteVars.colors.grey[500],
+        color: siteVars.colors.white,
+      })}
```

Comment:
These changes look very strange to me: without additional styling `circular` is useless now. May be we can use `bordered` there instead of variables:

```
<Icon bordered circular name="chess rook" />
```

Should mention that SUI provides a shadow by default:
https://semantic-ui.com/elements/icon.html#bordered
https://semantic-ui.com/elements/icon.html#circular

Reply:
For the examples we have the `bordered` example that shows the combination with `circular` as well. I can change the examples somehow (I also don't like the fact that we are using variables, but otherwise we cannot show  what circular stands for). The thing we need to address is that, in Teams theme, circular may need to appear without border (that's my understand from the created issue).

Reply:
Maybe we should altogether remove the example of the circular and use the border example to show how it can have rectangular and circular borders.

Reply:
share the same concerns as the ones that are expressed by @layershifter, but at the same time would rather like to leave `circular` example in docs.

The approach with `variables` that is currently taken could be problematic also in the following aspect - as now we haven't generalized variables' schema (in a way that some variables would be _guaranteed_ to introduced by any theme), we cannot be absolutely sure that each theme will have these variables used in the example, as well as that these variables will introduce expected effect.

What if, for now (maybe before the API of `Icon` is extended) we would introduce the following markup for this example:

```jsx
// to emphasize icon's boundaries 
const withShadow = { boxShadow: '0px 0px 5px grey' }

...
    <Icon
      name="chess rook"
      circular
      style={withShadow}
  />
```

You might argue that we would use inline styles here, but with variables, essentially, we would do the same. The difference this approach introduces is its simplicity, readability, as well as that we will be absolutely sure about produced effect of this style - and won't rely on colors defined by theme.

In future, however, we might think to extend Icon's API and adjust example's code accordingly. What do you, folks, think?

Reply:
I would vote for using the `boredered` prop, and we can update it later. We are saying all the time not use style, so I wouldn't like us to have it in the examples. Will revert the bordered example as it was before.

Reply:
great, totally agree - thank you!